src/app/src/main/kotlin/org/imperial/mrc/hint/exceptions/AdrException.kt

```diff
@@ -0,0 +1,9 @@
+package org.imperial.mrc.hint.exceptions
+
+import org.springframework.http.HttpStatus
+
+class AdrException(
+    key: String,
+    status: HttpStatus,
+    url: Array<String> = arrayOf("")
```

Comment:
codecov 👎  line is obviously tested.

Reply:
Is it because none of the tests test the default url parameter - they all provide it in some form I think, via `expectTranslatedAdrException`? But I don't think you should need the default anyway..?